source/.bazelrc

```diff
@@ -21,10 +21,22 @@ build --cxxopt='-D_GLIBCXX_USE_CXX11_ABI=0'
 build --extra_toolchains=@bazel_tools//tools/python:autodetecting_toolchain_nonstrict
 
 # Build with coverage info
-build:coverage --copt='-fprofile-instr-generate' --copt='-fcoverage-mapping' --linkopt='-fprofile-instr-generate' --linkopt='-fcoverage-mapping'
+build:coverage --copt='-fprofile-instr-generate' --copt='-fcoverage-mapping' --linkopt='-fprofile-instr-generate' --linkopt='-fcoverage-mapping' --run_under=//tools/coverage:run_under
+
+# Java coverage settings
+build:coverage --collect_code_coverage --instrumentation_filter='//neuropod/bindings/java:neuropod_java(_jar)?'
```

Comment:
Interesting, it doesn't need:

coverage_report_generator=@bazel_tools//tools/test/CoverageOutputGenerator/java/com/google/devtools/coverageoutputgenerator:Main

I guess this is because of upraded bazel. Anyway, if it works, this is great that this is optional now.